lib/middleware/authentication/authentication.test.js

```diff
@@ -28,14 +28,11 @@ describe('authentication', () => {
     beforeAll(() => {
       process.env.NODE_ENV = 'production'
       process.env.USE_AUTH = 'true'
+      delete process.env.USERNAME
```

Comment:
Moved this to ensure that the username/password is always reset in all the tests.